sopel/config/core_section.py

```diff
@@ -1181,12 +1181,13 @@ def homedir(self):
 
     :default: ``PLAIN``
 
-    ``EXTERNAL`` is also supported, e.g. for using :attr:`client_cert_file` to
-    authenticate via CertFP.
+    ``EXTERNAL`` is supported, e.g. for using :attr:`client_cert_file` to
+    authenticate via CertFP, and ``SCRAM-SHA-256`` for challenge-response
+    authentication.
```

Comment:
Can we turn this into something more like:

```suggestion
    Supported mechanisms are:

        * ``PLAIN`` — for basic password-based SASL; the default
        * ``EXTERNAL`` — for using :attr:`client_cert_file` to
          authenticate via CertFP
        * ``SCRAM-SHA-256`` — for challenge-response authentication
```

(Have not spent any time trying to confirm line lengths or exact wrapping/indents in the GitHub comment-box.)

Reply:
Much more clear, thanks.